### DISCUSSION TITLE
NGINX issues on production

### DISCUSSION THREAD

**Discussion by sajkamal** [supabase/supabase#1714] [General]
Hello everyone, 

When we push out our supbase app to production. We are using NGINX to serve our app. We are running into complications of url used to initialize the supabase client. We have modified it to the name of the container services. Its still not reachable from a public domain. 

How did everyone else solve it?

![image (13)](https://user-images.githubusercontent.com/1057846/119611621-9cd02c80-be03-11eb-9f8b-57ad061f8f00.png)

**Comment by soedirgo** [comment]
Can you try changing the URL (first parameter) for supabase-js to your public-facing endpoint for the Docker setup's API gateway? For the record here are my repro steps:
- `supabase eject` with the latest version (v0.4.0) of the CLI, default ports,
- start the Supabase Docker setup with `docker compose -f docker/docker-compose.yml up -d`,
- serve a public-facing endpoint with `ngrok`,
- run the following NodeJS script:
```js
const { createClient } = require('@supabase/supabase-js')
const supabase = createClient('http://<...>.ngrok.io', '<anon-key>')
supabase.auth.signUp({ email: 'a@example.com', password: 'abcdef' }).then(console.log)
```
